pkg/coreos/actuator_reconcile.go

```diff
@@ -97,6 +97,31 @@ func (c *actuator) cloudConfigFromOperatingSystemConfig(ctx context.Context, con
 			})
 		}
 
+		if unit.Name == "containerd.service" && config.Spec.EnableContainerD {
+			containerdUnit, err := c.box.Find("containerd-exec-start")
+			if err != nil {
+				return "", nil, fmt.Errorf("could not generate cloud config: %v", err)
+			}
+			u.DropIns = append(u.DropIns, UnitDropIn{
+				Name:    "11-exec-start-config.conf",
+				Content: string(containerdUnit),
+			})
+		}
+
+		cloudConfig.CoreOS.Units = append(cloudConfig.CoreOS.Units, u)
+	}
+
+	if config.Spec.Purpose == extensionsv1alpha1.OperatingSystemConfigPurposeReconcile && config.Spec.EnableContainerD {
+		u := Unit{Name: "docker.service"}
```

Comment:
Same here, we probably should add a constant.

Reply:
Done